src/Apps/Collect2/Routes/Collection/Components/CollectionsHubRails/ArtistSeriesRail/ArtistSeriesEntity.tsx

```diff
@@ -128,8 +130,11 @@ const SingleImgContainer = styled(Box)`
   }
 `
 
-const CollectionTitle = styled(Serif)`
-  width: 100%;
+const CollectionTitle = styled(Serif)<{ isMobile: boolean }>`
```

Comment:
Is this something we could use the `<Truncator>` component for? 

https://github.com/artsy/reaction/blob/c5f3a8254436e75f9b09808a52c0ffd8c5cd567c/src/Components/Publishing/Sections/ArtworkCaption.tsx#L211

Reply:
Didn't even know this was here. I will use this component instead.